provider/common/bootstrap.go

```diff
@@ -250,6 +253,13 @@ func BootstrapInstance(
 		if zone == "" || environs.IsAvailabilityZoneIndependent(err) {
 			return nil, "", nil, errors.Annotate(err, "cannot start bootstrap instance")
 		}
+
+		select {
+		case <-ctx.Done():
+			return nil, "", nil, errors.Annotatef(err, "starting bootstrap instance (cancelled)")
```

Comment:
existing message, but let's change from "bootstrap instance" to "controller"

Reply:
Fixed in https://github.com/juju/juju/pull/12475